engine/classes/Elgg/EntityIconService.php

```diff
@@ -435,6 +435,11 @@ public function getSizes($entity_type = null, $entity_subtype = null, $type = 'i
 			$sizes = $this->hooks->trigger("entity:$type:sizes", $entity_type, $params, $sizes);
 		}
 
+		if (empty($sizes)) {
```

Comment:
It already is an array (unless hook handlers are doing something silly): https://github.com/juho-jaakkola/Elgg/blob/c01771685d79b478c8b778eea7367c9ca9f91fbe/engine/classes/Elgg/EntityIconService.php#L422